src/sql/workbench/contrib/executionPlan/browser/executionPlanEditor.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as DOM from 'vs/base/browser/dom';
+import { localize } from 'vs/nls';
+import { IEditorOpenContext } from 'vs/workbench/common/editor';
+import { EditorPane } from 'vs/workbench/browser/parts/editor/editorPane';
+import { ITelemetryService } from 'vs/platform/telemetry/common/telemetry';
+import { IThemeService } from 'vs/platform/theme/common/themeService';
+import { ExecutionPlanInput } from 'sql/workbench/contrib/executionPlan/common/executionPlanInput';
+import { CancellationToken } from 'vs/base/common/cancellation';
+import { IStorageService } from 'vs/platform/storage/common/storage';
+import { IEditorOptions } from 'vs/platform/editor/common/editor';
+import { ExecutionPlanView } from 'sql/workbench/contrib/executionPlan/browser/executionPlan';
+import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
+
+export class ExecutionPlanEditor extends EditorPane {
+
+	public static ID: string = 'workbench.editor.executionplan';
+	public static LABEL: string = localize('executionPlanEditor', "Execution Plan Editor");
```

Comment:
for user facing strings I say `Query Execution Plan Editor`, though not sure if this shows in UI anywhere.